assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -324,13 +325,13 @@ export default function ArchivePicker({view = ADMIN_VIEW}) {
       dispatch({type: ACTIONS.ADDED_IMAGE_TO_POST});
       await timeout(1000);
       dispatch({type: ACTIONS.CLOSE_SIDEBAR});
-      document.querySelector('.media-modal-close').click();
+      document.querySelector('.media-modal-close .media-modal-icon').click();
```

Comment:
I've tested without `.media-modal-icon` and seems to be working fine. Could you check it, please?

Reply:
Yes it works, but @mleray wanted it to be more specific. The class could be present in some other element.